airdrop-cli/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "ubq-airdrop-cli",
+  "version": "1.0.0",
+  "description": "",
+  "author": "Keyrxng",
+  "main": "dist/index.js",
+  "scripts": {
+    "start": "node dist/index.js",
+    "tally": "node dist/index.js tally",
+    "tally-multi-csv": "node dist/index.js tally-multi-csv",
+    "tally-from": "node dist/index.js tally-from",
+    "single": "node dist/index.js single",
+    "help": "node dist/index.js help",
+    "build": "npx tsc",
+    "test": "npx jest"
+  },
```

Comment:
Looks like I forgot to include as part of the spec to use https://github.com/ubiquity/ts-template

It sets up a lot of the project for you, but also includes some essential linting rules that will make your code higher quality. The easiest way to include it in your work now is to 
```sh
git merge --allow-unrelated-histories
```